rdt/transformers/categorical.py

```diff
@@ -33,6 +33,12 @@ class CategoricalTransformer(BaseTransformer):
             Defaults to ``False``.
     """
 
+    INPUT_TYPE = 'categorical'
+    OUTPUT_TYPES = {'value': 'float'}
+    DETERMINISTIC_TRANSFORM = False
```

Comment:
This should be made dynamically, since this depends on whether `fuzzy==True` or not.